src/nrncvode/netcvode.cpp

```diff
@@ -3065,8 +3064,7 @@ void PreSyn::deliver(double tt, NetCvode* ns, NrnThread* nt) {
             TQItem* q = ns->p[i].tq_->least();
             Cvode* cv = (Cvode*) q->data_;
             if (tt < cv->t_) {
-                int err = NVI_SUCCESS;
-                err = cv->handle_step(nrn_ensure_model_data_are_sorted(), ns, tt);
+                cv->handle_step(nrn_ensure_model_data_are_sorted(), ns, tt);
```

Comment:
We should write an error message here. Is anyone able to tell me what we should write?

Reply:
Doesn't seem like something a user could do anything about if they encountered it. Therefore, something semi generic would be sufficient?
```
"warning: cv->handle_step failed with err = {err}"
```
Not sure it fits the theme of removing unused variables.